blocks/extras/initialization.py

```diff
@@ -34,3 +34,19 @@ def make_matrix(size, perm):
             return make_matrix(shape[0], self.permutation)
         else:
             return make_matrix(shape[0], rng.permutation(shape[0]))
+
+
+class NormalizedInitialization(NdarrayInitialization):
+    u"""Initialize the parameters with the Glorot method, as introduced in
```

Comment:
We should probably get Scrutinizer set up on this repo even if we don't
have Travis.

On Mon, Jul 6, 2015 at 1:37 PM, dmitriy-serdyuk <notifications@github.com>
wrote:

> In blocks/extras/initialization.py
> <https://github.com/mila-udem/blocks-extras/pull/12#discussion_r33967301>:
>
> > @@ -34,3 +34,19 @@ def make_matrix(size, perm):
> >              return make_matrix(shape[0], self.permutation)
> >          else:
> >              return make_matrix(shape[0], rng.permutation(shape[0]))
> > +
> > +
> > +class NormalizedInitialization(NdarrayInitialization):
> > +    u"""Initialize the parameters with the Glorot method, as introduced in
>
> There is no scrutinizer in this repo, but the first line should be a short
> description. Your description doesn't fit in one line, so you'll have to
> add a bit more detailed paragraph after a blank line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/blocks-extras/pull/12/files#r33967301>.
>